airflow/__init__.py

```diff
@@ -54,18 +54,30 @@
 PY39 = sys.version_info >= (3, 9)
 PY310 = sys.version_info >= (3, 10)
 
+# Things to lazy import in form 'name': 'path.to.module'
+__lazy_imports = {
+    'DAG': 'airflow.models.dag',
+    'XComArg': 'airflow.models.xcom_arg',
+    'AirflowException': 'airflow.exceptions',
+}
+
 
 def __getattr__(name):
     # PEP-562: Lazy loaded attributes on python modules
-    if name == "DAG":
-        from airflow.models.dag import DAG
+    path = __lazy_imports.get(name)
+    if path:
+        import operator
 
-        return DAG
-    if name == "AirflowException":
-        from airflow.exceptions import AirflowException
+        # Strip of the "airflow." prefix because of how `__import__` works (it always returns the top level
+        # module)
+        without_prefix = path.split('.', 1)[-1]
```

Comment:
```suggestion
    # Strip of the "airflow." prefix because of how `__import__` works (it always returns the top level
    # module)
    without_prefix = path.split('.', 1)[-1]
```
nit cont.